src/pages/Rockets.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  fetchRockets,
+  bookedRocket,
+  cancelledRocket,
+} from '../redux/actions/rockets';
+
+const Rockets = () => {
+  // set dispatch
+  const dispatch = useDispatch();
+
+  // fetch rockets on page mount
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, [dispatch]);
```

Comment:
- [ ] when i navigate to mission page and come back to this page, all my booked rocket is changed to unbooked rocket, I guess it is because you run fetchRockets on every render, the fetchRockets should be called on initial render, this way you keep your state preserved